src/Entity/SettingInterface.php

```diff
@@ -47,9 +47,11 @@ public function getStrategy(): string;
     public function getType(): string;
 
     /**
-     * @return bool
+     * Possible values ['shared', 'local', 'sdk']
+     *
+     * @return string
      */
-    public function isProject(): bool;
+    public function getSettingType(): string;
```

Comment:
I'm not sure we can just rename the method in the contract. For such cases normally we should mark it as deprecated and introduce new methods. And later, in a couple of weeks/month, we can remove it with a major release

Reply:
There're no problems. I'll move it back with new methods